people-ops/travel-policy.html.md

```diff
@@ -75,9 +76,15 @@ Where possible, we'll prefer rooms with two single beds or two queen beds.
 
 ## Reimbursements
 
-To obtain a reimbursement, every team member has to submit a complete travel expense report,
-attaching every expense receipt (for both meals and travel) within seven days of their return
-date from the trip.
+To obtain reimbursement, each team member must use the American Express corporate credit card. Using
+the company credit card, it’s not necessary to make the expense report. For all expenses where the
```

Comment:
Transactions done through the company credit card don't require an expense report.